src/lib/components/chatInput/ModelPickerPopover/ModelPickerPopoverContent.svelte

```
@@ -0,0 +1,127 @@
+  const currentUser = useCurrentUser(null);
+</script>
+
+<Command.Root>
+  <Command.Input placeholder="Search model..." bind:ref={inputRef} />
+  <Command.List>
+    <Command.Empty>No model found.</Command.Empty>
+
+    {#if !BILLING_ENABLED || isSubscribed($currentUser.data?.data ?? null) || byokEnabled}
+      {@const viewModels = MODELS.filter((model) => byokEnabled || !MODEL_DETAILS[model].free)}
+      {@const highlightedModels = viewModels.filter((model) =>
+        HIGHLIGHTED_MODELS.includes(model as any),
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)